Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -422,7 +423,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 }
 
                 $fixEvent = $this->stopwatch->getEvent('fixFiles');
-                $output->writeln(sprintf('Fixed all files in %.3f seconds, %.3f MB memory used', $fixEvent->getDuration() / 1000, $fixEvent->getMemory() / 1024 / 1024));
+                $output->writeln(sprintf('Processed all files in %.3f seconds, %.3f MB memory used', $fixEvent->getDuration() / 1000, $fixEvent->getMemory() / 1024 / 1024));
+                if (!$showProgress) {
```

Comment:
Why not output these results when the process is shown as well? 
You can register 2 listeners, so your could always be added.